integration_tests/test_aggregator_batch_commands.py

```diff
@@ -40,18 +41,24 @@ def on_market_cycle(self, market_info):
                                                         energy=device_event["device_info"]["available_energy_kWh"] / 2) \
                     .list_offers(area_uuid=device_event["area_uuid"])
 
-                if "energy_requirement_kWh" in device_event["device_info"] and \
+            if "energy_requirement_kWh" in device_event["device_info"] and \
                         device_event["device_info"]["energy_requirement_kWh"] > 0.0:
                     self.add_to_batch_commands.bid_energy(area_uuid=device_event["area_uuid"], price=30,
                                                           energy=device_event["device_info"][
                                                                      "energy_requirement_kWh"] / 2) \
-                        .list_bids(area_uuid=device_event["area_uuid"]) \
-                        .last_market_stats(area_uuid=device_event["area_uuid"])
+                        .list_bids(area_uuid=device_event["area_uuid"])
+            self.add_to_batch_commands.grid_fees(area_uuid=self.redis_market.area_uuid, fee_cents_kwh=5)
+            self.add_to_batch_commands.last_market_dso_stats(self.redis_market.area_uuid)
+            self.add_to_batch_commands.last_market_stats(self.redis_market.area_uuid)
```

Comment:
```suggestion
                        .list_bids(area_uuid=device_event["area_uuid"])
                        .grid_fees(area_uuid=self.redis_market.area_uuid, fee_cents_kwh=5)
                        .last_market_dso_stats(self.redis_market.area_uuid)
                        .last_market_stats(self.redis_market.area_uuid)
```
Matter of preference, I like this JS-style method chaining a lot though.